deploy.js

```diff
@@ -33,7 +33,7 @@ const secrets = [
 async function main() {
   const buildCommands = [
     'rm -rf node_modules/webpack-sources',
-    'npm i webpack-sources@1.0.1 ',
```

Comment:
😄

Reply:
So much hacks and workarounds for a simple JS function :smile: